### PR TITLE
Microsoft Intune Modeling Rules

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to add a pack of modeling rules to Microsoft Intune.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [X] 8.2.0

### CODE REVIEW

File: Packs/MicrosoftIntune/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Note: The logs will be stored in the dataset named *msft_azure_raw*. 
+To filter a query to focus only on Microsoft Intune logs, use the following filters:
+- In XQL queries, use: *|filter _collector_name=<Name_of_Instance_from_Azure_Event_Hub_Configuration>*
+- In Datamodel queries, use: *|xdm.observer.name=<Name_of_Instance_from_Azure_Event_Hub_Configuration>*
+
+## Collect Events from Vendor
+
+In order to use the collector, you need to use the following option:
+
+- [Microsoft Intune](#microsoft-intune)
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftIntune/README.md:14]: I think these links should be removed.  The second link, links to the section it is in.  This Readme is not so large that we need to have these links.

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

File: Packs/MicrosoftIntune/ModelingRules/MicrosoftIntune_1_3/MicrosoftIntune_1_3.xif [hunk 2]
```diff
@@ -0,0 +1,57 @@
+[MODEL: dataset="msft_azure_raw"]
+filter _collector_name = "intune-xdr-eventhub (demistodev.onmicrosoft.com)" and category contains "DeviceComplianceOrg"
```
Comment (evisochek) [inline Packs/MicrosoftIntune/ModelingRules/MicrosoftIntune_1_3/MicrosoftIntune_1_3.xif:2]: demistodev.onmicrosoft.com this is demisto user name why you are using it in the filter?

Comment (nkanon) [reply Packs/MicrosoftIntune/ModelingRules/MicrosoftIntune_1_3/MicrosoftIntune_1_3.xif]: Fixed with the correct filter

**Review by evisochek — APPROVED** [review]

**Pull request merged**